Assets/MixedRealityToolkit.Providers/WindowsMixedReality/WindowsMixedRealityDeviceManager.cs

```diff
@@ -653,6 +653,8 @@ private BaseWindowsMixedRealitySource GetOrAddController(InteractionSource inter
                         // Return null so we don't raise the source detected.
                         return null;
                     }
+                    else
```

Comment:
nit: please wrap the following line in `{` `}`, which is the style we've been using even for single line cases